action/protocol/execution/evm/evmstatedbadapter.go

```diff
@@ -220,14 +241,11 @@ func (stateDB *StateDBAdapter) accountCreationOpts() []state.AccountCreationOpti
 // CreateAccount creates an account in iotx blockchain
 func (stateDB *StateDBAdapter) CreateAccount(evmAddr common.Address) {
 	addr, err := address.FromBytes(evmAddr.Bytes())
-	if err != nil {
-		log.L().Error("Failed to convert evm address.", zap.Error(err))
+	if stateDB.assertError(err, "Failed to convert evm address.", zap.Error(err)) {
```

Comment:
it will call `stateDB.logError(err)` in new behaviours, does it matters?

Reply:
it does not matter much. `stateDB.logError(err)` just updates the most recent error of EVM, does not affect consensus